modules/core/src/main/java/org/apache/ignite/internal/hlc/HybridClock.java

```diff
@@ -44,7 +51,16 @@ public interface HybridClock {
     HybridTimestamp now();
 
     /**
-     * Gets a current timestamp. It is a fast way to get timestamp because it doesn't have to tick the logical part of the clock.
+     * Creates a timestamp for new event. A timestamp is guarantied to be unique and monotonically grown and follow the causal.
+     *
+     * @param causal The causal timestamp.
+     *
+     * @return The hybrid timestamp.
+     */
+    HybridTimestamp now(HybridTimestamp causal);
```

Comment:
Do I understand correctly that this new method is logically equivalent to `update()` followed by `now()`? If so, is it worth adding at all? It is only used in one place, and the corresponding `nowLong()` is not used externally at all.

Reply:
This method was removed, as it (unintentionally) duplicates "update"